File: btjs_page/btjs/mod/checkers/mod/is_number.js


/*
This function is used to verify if 
the data sent is of numeric type, 
returning true if it is.
*/
// return bool
export function is_number(ANY) {
    if((Number.isInteger(ANY))
    || (ANY % 1 !== 0)) {
        return true
    }
    return false
}

File: demons/btpy/btpy_checkers/mod/is_number/is_number.py



def is_number(ANY_DATA:any) -> bool:
    """
    Function that returns true if 
    the data sent is float or 
    integer type.
    FINAL
    """
    if(not type(ANY_DATA) == int
    and not type(ANY_DATA) == float): 
        return False
    return True